frontend/cli/cmd_schema_diff.go

```diff
@@ -116,7 +116,7 @@ func localSchema(ctx context.Context, projectConfig projectconfig.Config, bindAl
 			if err != nil {
 				moduleSchemas <- either.RightOf[*schema.Module](err)
```

Comment:
BTW given that this Either is (T, error) you might want to use [Result](https://pkg.go.dev/github.com/alecthomas/types@v0.16.0/result) instead, it has a much nicer interface for this use case.

Reply:
oh sweet